res/anims/poses/alternate_oc_anims/naduwide-arc.bvh

```diff
@@ -0,0 +1,304 @@
+HIERARCHY
```

Comment:
This animation had one of the arms cross inside the body. Specifically the left arm went inside the chest while testing

Reply:
i know. Whenever I tried to fix, it got boked so i left it as is.